contents/blog/multi-product-pricing.md

```diff
@@ -0,0 +1,70 @@
+---
+date: 2023-04-04
+title: What we've learned about multi-product pricing (so far)
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author:
+  - james-hawkins
+featuredImage: ../images/blog/posthog-ceo-diary-blog.png
+featuredImageType: full
+category: CEO diaries
+---
+
+PostHog started as a single product company. We just sold product analytics with a bunch of large extra features thrown in – session replay, feature flagging, experimentation, and even a Customer Data Platform.
+
+We charged by event ingested. Obviously, we wound up with abuse – a handful of high volume users would send zero events, but would use our Javascript snippet to do just session replay, to store _millions_ of recordings.
+
+We decided a multi-product pricing strategy had long term advantages, and – to be frank – we had to stop people abusing recording, and we didn't want to limit it for users already paying us a healthy amount. This is what we've learned so far. 
+
+## 1. Usage increased when we started charging
+
+The most important thing, but the one we didn't anticipate, was charging for session replay very quickly led to more usage.
+
+Why? Positioning. 
+
+If you see two products listed side-by-side on a website, you know each is being treated as a first-class citizen. Beforehand, we were "product analytics++". That meant no one was taking our session replay feature seriously.
+
+The moment we started charging, awareness increased. Users took the feature more seriously, and they shared more demanding feedback – paying users are less bug tolerant!
+
+We put a small team on it, who quickly iterated with customers and improved the product. Session replay took off as a result – it now has similar daily usage to product analytics.
+
+## 2. Focusing on paying customers improves the free product (and drives growth)
+
+We spent the first year and a half of PostHog prioritizing our open-source project – we wanted to build an inbound sales model, so we could double down on product rather than investing in cold calling. Leverage, baby. This meant we'd need a great free product and a reasonably large community. After 18 months, we were ready to focus on our first paid product – product analytics.
+
+Our only company goal at this point was to get 5 reference, paying customers. By going really deep with this handful, we realized that we needed to improve our funnel experience – a core part of our product. We '[nailed funnels](new-vp-nailing-funnels)', and revenue started climbing as a result. However, delightfully, our _free_ user numbers also started growing much faster.
+
+This journey has since repeated itself with session replay. Since the free and paid session recording products are the same, except for a usage limit with the free product, focusing on more-demanding, paid users led to similar quality improvements for everyone.
+
+And, better yet, it's very easy to justify making improvements for paying customers, since it's long-term sustainable.
```

Comment:
```suggestion
This journey has repeated itself with session replay. 

Since the free and paid session replay products are largely the same, except for a usage limit with the free product, focusing on more demanding, paid users improved the product for everyone and drove word-of-mouth growth.

And, better yet, it's very easy to justify making improvements for paying customers, since it's long-term sustainable.
```

Reply:
@charlescook-ph Have tweaked this bit now.